src/components/RecipeCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { TimeIcon } from '@chakra-ui/icons';
+import { Text, Flex, Spacer } from '@chakra-ui/react';
+import Img from 'gatsby-image';
+import { graphql } from 'gatsby';
+
+interface Props {
+  data: GatsbyTypes.RecipeCardFragment;
+}
+
+function RecipeCard({ data }: Props): JSX.Element {
+  return (
+    <div style={{ marginRight: '10%', marginBottom: '10%' }}>
```

Comment:
Replace with Chakra `Box`

Reply:
It might look something like:
```suggestion
    <Box mr="10%" mb="10%">
```